### ISSUE TITLE
Following the Web Content Accessibility Guidelines (WCAG) for contrast

### ISSUE THREAD

**Issue by husamalhomsi** [LMMS/lmms.io#340]
I noticed some issues of low contrast on the website. Adequate contrast is necessary for all users, especially users with low vision. I think following the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/) for contrast would fix the issues and improve accessibility. Specifically, I think these criteria should be met:

- [Contrast (Minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum)
- [Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast)

This [article](https://webaim.org/articles/contrast/) helped me understand the criteria.

Some of the issues I noticed are described below; this is not a complete evaluation. I used this [checker](https://webaim.org/resources/contrastchecker/) to check contrast.

![image](https://user-images.githubusercontent.com/25503477/147892583-459b71ad-c5f6-4018-b23d-f4f14cbb6722.png)
This color combination on the homepage (and elsewhere) has a contrast ratio of 2.83:1 and fails Contrast (Minimum).
See the [check](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=12B148).
Making the background darker is the best fix.

![image](https://user-images.githubusercontent.com/25503477/147892987-75fa874d-a4b3-4021-99c3-f5cefc403b38.png)
This color combination on the homepage has a contrast ratio of 2.19:1 and fails Non-text Contrast.
See the [check](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=ED9F00).
Making the background darker is the easiest fix.

![image](https://user-images.githubusercontent.com/25503477/147894128-dc3cf838-f9a2-4926-aabf-9b0d50de04b7.png)
This color combination on the downloads page has a contrast ratio of 4.22:1 and fails Contrast (Minimum).
See the [check](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=7871C5).
Making the background darker is the best fix.

I tested making the backgrounds dark enough to meet the criteria; they look better!

Maybe the criteria should be required or recommended in the wiki.

**Comment by headquarter8302** [comment]
Hi, this repository has been given a streak of development recently. Can you check if there's still more of these color issues or if they've been rectified? I am interested in picking up this issue
